app/src/main/java/kth/jjve/xfran/models/UserProfile.java

```diff
@@ -1,5 +1,11 @@
 package kth.jjve.xfran.models;
 
+/*
+Function: defines userprofile object
+Activity:
+Jitse van Esch, Elisa Perini & Mariah Sabioni
+ */
+
```

Comment:
I really like this description template, we can use this as a standard

Reply:
Seconded, it's awesome, makes everything very clear